### PR TITLE
Add chip/tag background color token

### PR DESCRIPTION
**Pull request by fredvisser**
# Pull Request

## 🤨 Rationale

[Bug 2718212](https://dev.azure.com/ni/DevCentral/_workitems/edit/2718212): sl-details-item-keywords chip background color is too similar to section bg color in light theme

…requires this new Nimble token.

## 👩‍💻 Implementation

Ultimately used the same token values as `cardBorderColor`:
Light: 10% Black91
Dark: 10% Black15
Color: 10% White

## Tag preview

Screenshot 2024-04-26 at 11 56 36 AM
Screenshot 2024-04-26 at 11 59 58 AM
Screenshot 2024-04-26 at 12 00 46 PM

(Note, I'll fix the missing font-color token issue in my SystemLinkShared changes as well)

## 🧪 Testing

Visual review

## ✅ Checklist

- [x] I have updated the project documentation to reflect my changes or determined no changes are needed.

### CODE REVIEW

**Comment by fredvisser** [comment]
@NIbokeefe - Are these the correct color values for this use case (tags in the details panel)?

File: packages/nimble-components/src/theme-provider/design-token-comments.ts [hunk 1]
```diff
@@ -45,6 +45,7 @@ export const comments: { readonly [key in TokenName]: string } = {
     modalBackdropColor: 'Color of background overlay behind modal dialog boxes',
     popupBorderColor: 'Border color for menus and dialog boxes',
     cardBorderColor: 'Border color for cards',
+    chipBackgroundColor: 'Background color for chips',
```
Comment (rajsite) [inline packages/nimble-components/src/theme-provider/design-token-comments.ts:48]: Think I'd opt for `tagFillColor`

tag: openui working group resolved chip and pills as style variations of tags https://github.com/openui/open-ui/issues/266#issuecomment-1072843538

fill: seems more similar to a button type control and on buttons we refer to it as a fill

**Review by rajsite — APPROVED** [review]

File: change/@ni-nimble-components-0aeacb05-bfbf-47eb-8e8d-d47e996f3004.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (rajsite) [inline change/@ni-nimble-components-0aeacb05-bfbf-47eb-8e8d-d47e996f3004.json:1]: @fredvisser is this PR ready to merge or are you waiting for a response to this question? https://github.com/ni/nimble/pull/2040#issuecomment-2079904402

Comment (fredvisser) [reply change/@ni-nimble-components-0aeacb05-bfbf-47eb-8e8d-d47e996f3004.json:1]: Let's merge it - 🎉

**Comment by NIbokeefe** [comment]
> @NIbokeefe - Are these the correct color values for this use case (tags in the details panel)?

@fredvisser Yes, those are the correct chip color tags.  However we should sync about the shape of the chips which is not correct.

**Pull request merged**